src/test/kotlin/graphics/scenery/tests/examples/volumes/OutOfCoreRAIExample.kt

```diff
@@ -18,10 +18,10 @@ import net.imglib2.img.Img
 import net.imglib2.img.display.imagej.ImageJFunctions
 import net.imglib2.type.numeric.integer.UnsignedShortType
 import net.imglib2.type.volatiles.VolatileUnsignedShortType
-import org.janelia.saalfeldlab.n5.GzipCompression
-import org.janelia.saalfeldlab.n5.N5FSReader
-import org.janelia.saalfeldlab.n5.N5FSWriter
-import org.janelia.saalfeldlab.n5.imglib2.N5Utils
+//import org.janelia.saalfeldlab.n5.GzipCompression
+//import org.janelia.saalfeldlab.n5.N5FSReader
+//import org.janelia.saalfeldlab.n5.N5FSWriter
+//import org.janelia.saalfeldlab.n5.imglib2.N5Utils
```

Comment:
So, scenery only uses (used) n5 in tests? Please let me know if you find a good way of dealing with test-scope dependencies with JPMS. We have found it very challenging with SciJava Ops; we were unable to make a separate `module-info.java` in the `src/test` subtree work (see scijava/scijava@ae39206f8fe6eae2aa2da3b0557e0ecc003e1329 for details).